.github/workflows/ci-pytorch-dockers.yml

```diff
@@ -1,28 +1,14 @@
 name: Docker
 
 on:
-  push:
-    branches: [master, "release/*"]
-  pull_request:
-    branches: [master, "release/*"]
-    paths:
-      - "dockers/**"
-      - "!dockers/README.md"
-      - "requirements.txt"
-      - "requirements/*.txt"
-      - "requirements/pytorch/*"
-      - "environment.yml"
-      - ".github/workflows/*docker*.yml"
-      - "setup.py"
-  schedule:
-    - cron: "0 0 * * *"  # at the end of every day
+  pull_request: {}
```

Comment:
need to revert this later on :)

Reply:
I know, ready for it to be last commit :)